managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerFactoryImpl.java

```diff
@@ -512,6 +512,7 @@ public void shutdown() throws InterruptedException, ManagedLedgerException {
             ledger.asyncClose(new AsyncCallbacks.CloseCallback() {
                 @Override
                 public void closeComplete(Object ctx) {
+                    log.debug("[{}] Closed managed ledger.", ledger.getName());
```

Comment:
Same as above

Reply:
I have removed this change from this PR.